android/src/com/mapswithme/maps/widget/placepage/PlacePageView.java

```diff
@@ -29,6 +29,7 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.appcompat.widget.Toolbar;
+import androidx.core.view.GestureDetectorCompat;
```

Comment:
Look like unused import of `GestureDetectorCompat` class

Reply:
Hmm, AS added it automatically at some point.. I guess during my prior experiments.
I checked that it builds and works without it.
Thanks for spotting!